app/assets/javascripts/sprangular/services/account.coffee

```diff
@@ -52,9 +52,11 @@ Sprangular.service "Account", ($http, _, $q, Cart, Flash) ->
       $http.post '/spree/login.json', $.param params
         .success (data) ->
           service.populateAccount(data)
-          Flash.success 'Successfully signed in'
+          $translate('app.signed_in').then (paragraph) ->
+            Flash.success paragraph
         .error ->
-          Flash.error 'Sign in failed'
+          $translate('app.signin_failed').then (paragraph) ->
```

Comment:
hmm, not sure. might be thats its misreporting.

btw, I think we should put the translating inside the Flash service (since it always needed when calling Flash functions). Would save a lot of repetition.
something like:
```
Sprangular.factory 'Flash', ($timeout, translate) ->
  ...

  add: (type, message) ->
    $translate(message).then (translated) ->
      flash = type: type, text: translated
   
      @messages.push(flash)
      @timeout(flash)
```